docs/develop/func/cookbook.md

```diff
@@ -430,10 +437,10 @@ send_raw_message(msg, 3); ;; mode 3 - pay fees separately and ignore errors
 
 ### How to contain a body as ref to an internal message cell
 
-TODO: please provide some description like in a `Modulo operations` section
+In the body of a message that follows flags and other technical data, we can send `int`, `slice`, and `cell`. In the case of the latter, it is necessary to set the bit to `1` before `store_ref()` to indicate that the `cell` will go on.
```

Comment:
The description is a bit misleading. I believe it's better to say "we can send message body in the same cell as header, if we believe there will be enough place, or in the separate cell. The first option requires to add bit `0`, the second - `1`".